packages/vitest-pool-workers/src/pool/config.ts

```diff
@@ -50,6 +50,7 @@ const WorkersPoolOptionsSchema = z.object({
 	miniflare: z
 		.object({
 			workers: z.array(z.object({}).passthrough()).optional(),
+			experimentalImagesLocalMode: z.oboolean(),
```

Comment:
Not sure if this is the best place for this to live - it could just as well be somewhere else